source/js/store/index.js

```diff
@@ -1,17 +1,17 @@
 import thunkMiddleware from 'redux-thunk';
-import {createLogger} from 'redux-logger';
+// import {createLogger} from 'redux-logger';
```

Comment:
Consider cleaning up the commented out items, if you are no longer using them

Reply:
Good catch, thanks